packages/rich-text/package.json

```diff
@@ -66,6 +67,7 @@
     "@babel/preset-env": "7.12.11",
     "@babel/preset-react": "7.13.13",
     "@contentful/field-editor-test-utils": "^1.0.0",
-    "@contentful/rich-text-react-renderer": "^15.6.2"
+    "@contentful/rich-text-react-renderer": "^15.6.2",
+    "@udecode/plate": "^7.0.3"
```

Comment:
I needed this for `createEditorPlugins`. Anyone against importing other plugins from `plate` as well? cc @contentful/team-shelley